rosetta-source/src/main/rosetta/product-common-settlement-type.rosetta

```diff
@@ -372,6 +372,7 @@ type SettlementBase: <"A base class to be extended by the SettlementTerms class.
 		[metadata scheme]
 		[docReference ICMA GMRA namingConvention "Contractual Currency"
 		    provision "As defined in GMRA paragraph 2(k)/ paragraph 7(a) All the payments made in respect of the Purchase Price or the Repurchase Price of any Transaction shall be made in the currency of the Purchase Price (the Contractual Currency) save as provided in paragraph 10(d)(ii). Notwithstanding the foregoing, the payee of any money may, at its option, accept tender thereof in any other currency, provided, however, that, to the extent permitted by applicable law, the obligation of the payer to pay such money will be discharged only to the extent of the amount of the Contractual Currency that such payee may, consistent with normal banking procedures, purchase with such other currency (after deduction of any premium and costs of exchange) for delivery within the customary delivery period for spot transactions in respect of the relevant currency."]
+    nonSettlementReferenceCurrency string (0..1) <"The currency in which the swap stream is denominated. This is applicable for Non Deliverable Swaps where the currency of the swap cannot be used for settlement purposes.">
```

Comment:
This additional attribute is not required in the CDM.  The base currency for each InterestRatePayout can already be obtained from the following path.
TradeState -> trade -> tradableProduct -> product -> contractualProduct -> economicTerms -> payout -> interestRatePayout -> priceQuantity -> quantitySchedule -> unit